drivers/sensors/mpu60x0.c

```diff
@@ -404,7 +404,7 @@ static int __mpu_read_reg_i2c(FAR struct mpu_dev_s *dev,
   msg[0].frequency = CONFIG_MPU60X0_I2C_FREQ;
   msg[0].addr      = dev->config.addr;
   msg[0].flags     = I2C_M_NOSTOP;
-  msg[0].buffer    = &reg_addr;
+  msg[0].buffer    = (uint8_t *)&reg_addr;
```

Comment:
it's better to change reg_addr type to uint8_t since mpu_regaddr_e may have 4bytes length and the cast only work for little endian machine.

Reply:
done